ROADMAP.md

```diff
@@ -0,0 +1,226 @@
+  * "Blocking vs. non-blocking."
+  * Syscall documentation, to be used primarily by FS, but throughout the codebase.
+  * Internal: Node Architecture
+  * Internal: Initialization process
+  * Internal: Timers
+  * Internal: Event loop
+  * Internal: Signals
+  * Internal: Docs WG Process — **@chrisdickinson**
+* Guides:
+  * Internal: New Collaborator guide — **@nodejs/inclusivity**, **@ashleygwilliams** [nodejs/inclusivity#96](https://github.com/nodejs/inclusivity/issues/96)
+  * Internal: Move "cutting releases" into these guides.
+  * Streams: For Authors
+  * Streams: For Consumers
+  * Walkthroughs for each module
+* Reference:
+  * Glossary of terms, to be used by all docs.
+  * Streams: remove guide content from API doc once guides have been written.
+  * Streams: Descriptive spec — **@nodejs/streams** [nodejs/readable-stream#181](https://github.com/nodejs/readable-stream/issues/181)
+* Identifying other docs to create 
+
+#### :three: Improving API Metadata
+
+One of the most common requests we've received is that the API docs start
+including relevant version information alongside methods. This information
+should include the version the API was introduced in, when it was last changed,
+and when it was deprecated, if applicable. Second to that, we've received
+requests to automatically link types of parameters to the appropriate MDN or
+Node documentation sections, and note what (if any) errors an API will
+generate, and how it will propagate them.
+
+This points to the need to standardize tooling around this metadata, and then
+execute against that tooling.
+
+This task should track the work being done in task #1 — that is to say, this
+metadata should be tracked by remark plugins in as much as is possible.
+
+##### Subtasks
+
+* Per-section YAML — **@qard**, **@tflanagan** [nodejs/node#3867](https://github.com/nodejs/node/pull/3867)
+* Noting 'version introduced' on each API — **@tflanagan**
+  * Ideally this should link to the CHANGELOG for that release.
+* Standardizing API method signature documentation
+  * Type annotation — **@fansworld-claudio** [nodejs/node#4741](https://github.com/nodejs/node/pull/4741)
+    * (This will have to be brought into the Remark work as well.)
+  * Error generation
+* Automatically linking `syscall(2)`-format terms to the appropriate docs.
+* Allow authors to pick a simpler anchor for headings in addition to the
+  autogenerated anchor.
+
+### Notably Missing
+
+There are a few things notably missing from this list — this is not because
+they are not important to us, but because we are primarily constrained by time
+and the size of the content we are working within. Foremost among the missing
+pieces is an internationalization strategy. This task cannot be approached
+lightly, and will likely include touching *all* of the docs in a single PR.
+While we welcome discussion on how to approach this issue, we will not be
+executing on it until after March 2016. Likewise, search is an extant issue
```

Comment:
Totally not opposed if someone wants to do the legwork on this in advance. I had initially imagined we'd use something like Sphinx's precompiled JS search, but am open to alternatives (keeping in mind that the doc HTML is downloadable and usable separately from the website.)